frontend/src/components/layout/topmessage/HolidayPromoBanner.tsx

```diff
@@ -0,0 +1,72 @@
+import Link from "next/link";
+import { useL10n } from "../../../hooks/l10n";
+import { Localized } from "../../Localized";
+import styles from "./HolidayPromoBanner.module.scss";
+import { RuntimeData } from "../../../hooks/api/runtimeData";
+import {
+  getPeriodicalPremiumSubscribeLink,
+  isPeriodicalPremiumAvailableInCountry,
+} from "../../../functions/getPlan";
+import { ProfileData } from "../../../hooks/api/profile";
+import { useEffect, useState } from "react";
+
+type Props = {
+  profile?: ProfileData;
+  runtimeData?: RuntimeData;
+};
+
+export const HolidayPromoBanner = (props: Props) => {
+  const l10n = useL10n();
+  const [loadingProfile, setLoadingProfile] = useState(true);
+  const coupon = "HOLIDAY20";
+  const couponExpiry = "Dec 31, 2023";
```

Comment:
This won't work for localisation; that's not how dates are formatted (or abbreviated) in every locale. You could use [`Intl.DateTimeFormat`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat), but I'd actually recommend just "hardcoding" the date in the string, and let the localisers format it in a way that makes sense for their locale. It's not like we'll be changing this date, right?

Reply:
Originally did this incase we ever re use this banner in the future, we can re use the l10n string as well by just changing this variable. 

Hardcoded it into l10n now instead though.